comid/measurement_test.go

```diff
@@ -129,6 +129,16 @@ var (
 	goldenMKey1 uint64 = 120
 )
 
+func TestUIntMkey_NotSet(t *testing.T) {
+	meas := &Measurement{}
+	mkey := &Mkey{}
+	meas.Key = mkey
+	expectedErr := "unknown measurement key type: <nil>"
+	err := meas.Key.Valid()
+	require.NotNil(t, err)
+	assert.EqualError(t, err, expectedErr)
+}
+
 func TestUIntMkey_ToCBOR_ok(t *testing.T) {
```

Comment:
we don't seem to be testing the `ToCBOR` method here, why is the test called like that?

Reply:
Corrected to use the right naming of Marshal and UnMarshal